README.md

```diff
@@ -19,6 +19,13 @@ The `design-system-dashboard-cli` package is used to gather metrics on design sy
 
 # Running Build via Storybook
 
+To install dependencies in the component packages and run a local build of the component library in Storybook, run the following in the command line:
+```
+npm run dev
```

Comment:
> [!Note]
> This can be replaced with `yarn install && yarn dev` if we prefer. I just went with npm because it did not require a separate install command. 

```suggestion
yarn install && yarn dev
```